sky/utils/env_options.py

```diff
@@ -5,17 +5,27 @@
 
 class Options(enum.Enum):
     """Environment variables for SkyPilot."""
-    IS_DEVELOPER = 'SKYPILOT_DEV'
-    SHOW_DEBUG_INFO = 'SKYPILOT_DEBUG'
-    DISABLE_LOGGING = 'SKYPILOT_DISABLE_USAGE_COLLECTION'
-    MINIMIZE_LOGGING = 'SKYPILOT_MINIMIZE_LOGGING'
+
+    # (env var name, default value)
+    IS_DEVELOPER = ('SKYPILOT_DEV', False)
+    SHOW_DEBUG_INFO = ('SKYPILOT_DEBUG', False)
+    DISABLE_LOGGING = ('SKYPILOT_DISABLE_USAGE_COLLECTION', False)
+    MINIMIZE_LOGGING = ('SKYPILOT_MINIMIZE_LOGGING', True)
```

Comment:
We should show still show the stamps and file name/lines when `SKYPILOT_DEBUG` is set.

Reply:
Good catch, done. PTAL.